README.md

```diff
@@ -111,7 +111,7 @@ You can tweak the generated parser with several options:
   `peg.generate`
 - `--extra-options-file` — file with additional options (in JSON format) to
   pass to `peg.generate`
-- `--format` — format of the generated parser: `amd`, `commonjs`, `globals`,
+- `--format` — format of the generated parser: `amd`, `commonjs`, `es`, `globals`,
```

Comment:
i request that this say es2015, because there's an older forgotten es4x module format, and because it's likely that es9 or es10 will deliver an incompatible module format that exposes types and directly solves the michael jacksonscript problem

this also leaves it in parity with typescript, rollup, parcel, and rome

Reply:
I'm ok with that.  I'd like to document `es2015` or whatever we pick, but leave `es` as an alias, in case anyone started using it.

Reply:
Let's use #71 to track this since it's open and it's close enough to this discussion.  Please put your comment there?

Reply:
done